forge-gui/res/editions/Alchemy Horizons Baldur's Gate.txt

```diff
@@ -337,3 +328,32 @@ A239 U A-Krydle of Baldur's Gate @Bryan Sola
 A243 M A-Minsc & Boo, Timeless Heroes @Andreas Zafiratos
 A259 C A-Lantern of Revealing @Eytan Zana
 A262 U A-Navigation Orb @Robin Olausson
+
+[conjured]
+926 R Lightning Bolt @Irina Nordsol
+927 C Naturalize @Tim Hildebrandt
+928 C Plummet @Alix Branwyn
+45a R Hag of Ceaseless Torment @Konstantin Porubov
```

Comment:
The hags are unique cards that don't exist anywhere else, do the images still work correctly?

Reply:
Yes, I have tested to play "The Hourglass Coven" card, and all the "hag" I have conjured display correctly.
I have added the "conjured" section in the known section in file CardEdition.java, so they are loaded correctly